xchange-latoken/src/main/java/org/knowm/xchange/latoken/LatokenExchange.java

```diff
@@ -0,0 +1,148 @@
+package org.knowm.xchange.latoken;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import org.knowm.xchange.BaseExchange;
+import org.knowm.xchange.ExchangeSpecification;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.meta.CurrencyMetaData;
+import org.knowm.xchange.dto.meta.CurrencyPairMetaData;
+import org.knowm.xchange.exceptions.ExchangeException;
+import org.knowm.xchange.latoken.dto.LatokenException;
+import org.knowm.xchange.latoken.dto.exchangeinfo.LatokenCurrency;
+import org.knowm.xchange.latoken.dto.exchangeinfo.LatokenPair;
+import org.knowm.xchange.latoken.service.LatokenAccountService;
+import org.knowm.xchange.latoken.service.LatokenMarketDataService;
+import org.knowm.xchange.latoken.service.LatokenTradeService;
+import org.knowm.xchange.utils.AuthUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import si.mazi.rescu.RestProxyFactory;
+import si.mazi.rescu.SynchronizedValueFactory;
+
+public class LatokenExchange extends BaseExchange {
+
+  private static final Logger LOG = LoggerFactory.getLogger(LatokenExchange.class);
+  private static final int PRECISION = 8;
+
+  public static final String sslUri = "https://api.latoken.com";
```

Comment:
replace this with

```
private LatokenAuthenticated latoken;
```

Reply:
I am not sure what you mean... Can you further explain?

Reply:
please read my comments below and you'll understand why `String sslUri` is not needed here but `LatokenAuthenticated latoken` is